external-crates/move/crates/move-analyzer/trace-adapter/src/adapter.ts

```diff
@@ -148,7 +149,10 @@ export class MoveDebugSession extends LoggingDebugSession {
         this.sendEvent(new InitializedEvent());
     }
 
-    protected async launchRequest(response: DebugProtocol.LaunchResponse, args: ILaunchRequestArguments): Promise<void> {
+    protected async launchRequest(
```

Comment:
This (and the following) formatting changes were not necessary to achieve the stated goal of this PR but will make the code look better (as a side-note, I don't like TS auto-formatter not formatting these automatically)